### ISSUE TITLE
[🐛 Bug]: Missing types for the UiAutomator2 namespaced capabilities to use with Appium 2

### ISSUE THREAD

**Issue by mmmguitar** [webdriverio/webdriverio#10025]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

latest

### Node.js Version

latest

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
{
    "appium:optionalIntentArguments": "-d http://foobar.com",
    "appium:autoGrantPermissions": true,
}
```

### What happened?

I am writing Android appium tests using wdio + UiAutomator2 and am unable to configure any of the Android specific capabilities unless I ignore or extend the types locally myself.  The reason is because the types in wdio Capabilities.ts are missing.

I'm no expert here, but I think the move in Appium 2 for driver plugins means to support the W3C webdriver spec all extension capabilities must be namespaced with 'appium:', i.e. the capabilities for the UiAutomator 2 driver, and this is whats missing.

What currently exists is a very similar set of capabilities with no namespace on the interface 'AppiumAndroidCapabilities'.  These I believe are options for Appium 1 and their use yield runtime wdio errors with Appium 2 given the lack of namespace.  

I see this was an issue for iOS with AppiumIOSCapabilities + XCUITest in https://github.com/webdriverio/webdriverio/issues/7843.  There was a merged PR to fix the issue https://github.com/webdriverio/webdriverio/pull/7844, its the same problem just was only fixed for iOS.

The new UiAutomater2 capabilities are here, https://github.com/appium/appium-uiautomator2-driver.

### What is your expected behavior?

I should be able to set andoird capabilities for the UiAutomator2 driver as defined at https://github.com/appium/appium-uiautomator2-driver and the types should be valid.

### How to reproduce the bug.

Trying to set any Android specific capabilities using Appium 2, UiAutomater2 will result in type errors.

Removing the name space + using the Appium 1 option types defined  yields a wdio error at runtime given its missing the namespace.

### Relevant log output

```typescript
If you use the capabilities without 'appium:'  that have valid types you get a runtime wdio error:

Error: Invalid or unsupported WebDriver capabilities found ("autoGrantPermissions"). Ensure to only use valid W3C WebDriver capabilities (see https://w3c.github.io/webdriver/#capabilities).If you run your tests on a remote vendor, like Sauce Labs or BrowserStack, make sure that you put them into vendor specific capabilities, e.g. "sauce:options" or "bstack:options". Please reach out to to your vendor support team if you have further questions.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
Thanks for reporting!

Mind raising a PR to extend the Appium Capabilities in [`Capabilities.ts`](https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-types/src/Capabilities.ts)?

Any contributions that resolves the bug are highly appreciated. Please take a look into our [contribution guidelines](https://webdriver.io/docs/contribute) and let us know if you have any questions. Cheers!

**Comment by mmmguitar** [comment]
Thanks, good to hear it may be warmly received.  Yeah I could take a look, thanks for the guidelines.

**Comment by christian-bromann** [comment]
> Trying to set any Android specific capabilities using Appium 2, UiAutomater2 will result in type errors.

This was fixed in #10317, please verify that you should be able to use `appium:autoGrantPermissions` as capability now. Happy to reopen if the error still occurs.

**Comment by mmmguitar** [comment]
Yeah I updated to 8.10 and the types I've used appear to work just fine now.  That #10317 was alot more involved than my initial attempt.  Many thanks!
